viewer/apiSessions.js

```diff
@@ -487,6 +487,9 @@ module.exports = (Config, Db, internals, molochparser, Pcap, version, ViewerUtil
     req.query.line = req.query.line || false;
     req.query.base = req.query.base || 'ascii';
     req.query.showFrames = req.query.showFrames === 'true' || false;
+    // displaying images and uncompressing require all the packets from a session
+    // *2 because the packets can be out of order, we truncate again in the reassemble call
+    req.query.packets = req.query.needimage || req.query.needgzip ? 10000 : +req.query.packets * 2;
```

Comment:
* req.query.packets doesn't have to be set, so you probably should first set to the default of 200.
* the *2 shouldn't be here since req.query.packets is what it is truncated to. I think you should just do req.query.packets*2 down below so req.query.packets*2, 10);

Reply:
ah ok, I won't modify req.query.packets 🤭